azure-pipelines.yml

```
@@ -91,6 +91,10 @@ jobs:
   - bash: ci/scripts/clang-format.sh $SYSTEM_PULLREQUEST_TARGETBRANCH
     condition: eq(variables['Build.Reason'], 'PullRequest')
     displayName: clang-format (C/C++ lint)
+    # Use json schema to check testplans formatting
+  - bash: ci/scripts/testplans-lint.sh
+    condition: eq(variables['Build.Reason'], 'PullRequest')
+    displayName: testplans lint (json schema)
```

I think it's better to say the intention instead of implementation
```suggestion
    displayName: Validate testplans against schema
```

I general I think a lot of occurance of "lint" in this PR actually means validate.

---

I agree with your comment about intention vs implementation, I'll make that change.

I also agree with your suggestion about the semantics of `lint` vs `validate` - what we are doing with the JSON schema here is a lot closer to validation than it is linting. Do you want me to just update the comments / display names, or do you think its worth replacing all references to `lint` in this PR with `validate` or some equivalent, for the sake of readability?

---

Ideally replace all references/file names.